src/pages/LearnPage.tsx

```diff
@@ -64,101 +59,96 @@ function LessonGroup({ title, children }: LessonGroupProps) {
   )
 }
 
-const dp = await fetchLessonContent('dynamic_programming/lesson.md')
-const markdown1 = await fetchLessonContent('dynamic_programming/markdown1.md')
-const markdown2 = await fetchLessonContent('dynamic_programming/markdown2.md')
-const markdown3 = await fetchLessonContent('dynamic_programming/markdown3.md')
-
-const q1 =
-  'Which of the problems below can be solved effectively using dynamic programming.'
-const ans1 = ['Coin denomination', 'Knapsack Problem', 'Sorting List']
-const correct1 = ['Coin denomination', 'Knapsack Problem']
+interface lesson {
+  group: string
+  name: string
+  urlParam: string
+}
 
-const q2 =
-  'In dynamic programming, what is the key characteristic that distinguishes it from other algorithmic approaches?'
-const ans2 = ['Recursion', 'Iteration', 'Memoization']
-const correct2 = ['Memoization']
+const lessons: lesson[] = [
+  {
+    group: 'Intro Algo Design',
+    name: 'Brute Force',
+    urlParam: 'dp',
+  },
+  {
+    group: 'Intro Algo Design',
+    name: 'Intro Greedy',
+    urlParam: 'greedy',
+  },
+  {
+    group: 'Intro Algo Design',
+    name: 'Time & Memory Analysis',
+    urlParam: 'dp',
+  },
+  {
+    group: 'Intro Data Structures',
+    name: 'Prefix Sums',
+    urlParam: 'dp',
+  },
+  {
+    group: 'Intro Data Structures',
+    name: 'Lists & Vectors',
+    urlParam: 'dp',
+  },
+  {
+    group: 'Intro Data Structures',
+    name: 'Stacks & Queues',
+    urlParam: 'dp',
+  },
+]
```

Comment:
we need to discuss where this metadata should live, but I think this is fine for now

Reply:
yes, we can next meeting